tools/site_cobble/rdl_pkg/utils.py

```diff
@@ -7,4 +7,15 @@ def to_camel_case(template_string, uppercamel=False):
 
 
 def to_snake_case(template_string):
-    return inflection.underscore(template_string)
\ No newline at end of file
+    return inflection.underscore(template_string)
+
+def vhdl_bitstring(template_string, size):
+    val = int(template_string,0)
+    return '"{0:0{1}b}"'.format(val,size)
+
+def vhdl_2k8bitstring(template_string, size):
```

Comment:
I have a small preference for this to be named `vhdl_2008_bitstring` for the sake of clarity.